packages/block-library/src/navigation-link/edit.js

```diff
@@ -304,28 +296,41 @@ function NavigationLinkEdit( {
 	);
 }
 
+// function getRGB
 export default compose( [
 	withSelect( ( select, ownProps ) => {
 		const {
 			getBlockAttributes,
 			getBlockParents,
 			getClientIdsOfDescendants,
 			hasSelectedInnerBlock,
+			getColorObjectByColorSlug,
 		} = select( 'core/block-editor' );
 		const { clientId } = ownProps;
 		const rootBlock = getBlockParents( clientId )[ 0 ];
```

Comment:
Is this always guaranteed to be the parent Navigation Block? What if the Nav Block were within a Columns Block for example? I believe it would then traverse up the tree and the rootBlock would be the Columns Block...

Reply:
Sorry, this marked the wrong code spot. I mean to reference

```
const rootBlock = getBlockParents( clientId )[ 0 ];
```

Reply:
Now solved by getting parents filtered by the block name in #20032.